### DISCUSSION TITLE
Quicksilver Mainnet

### DISCUSSION THREAD

**Discussion by 0xCryptoPunk** [nodesxploit/testnet#6] [Q&A] [answered]
Kapan guide untuk Quikcsilver mainnet bang?

**Comment by nodexone** [comment, marked as answer]
untuk saat ini belum dibuat bre, tunggu ya satu satu wkwk

**Reply by 0xCryptoPunk** [reply]
oke bang siap, seklalian yang lain juga bang tolong di update ya

**Reply by nodexone** [reply]
oke bang siapp, makasih supportnya ya
